index.js

```diff
@@ -1,13 +1,130 @@
-const homeMenu = document.getElementById('humMenu');
 const navCo = document.getElementById('nav_list_container');
 const closeBtn = document.getElementById('close-btn');
+const homeMenu = document.getElementById('humMenu');
 const body = document.getElementById('body');
 const homeBtn = document.getElementById('home-btn');
 const projectsBtn = document.getElementById('projects-btn');
 const aboutBtn = document.getElementById('about-btn');
 const contactBtn = document.getElementById('contact-btn');
 const headerDesign = document.getElementById('shape2');
 const shape1 = document.getElementById('shape1');
+const popUpWindow = document.getElementById('popup-window');
+const popupCloseBtn = document.getElementById('popup-close-btn');
+const card = document.querySelectorAll('.card');
+
+const projectData = [
+  {
+    seeProjectBtn: document.getElementById('project-btn1'),
+    name: 'Project name goes here',
+    description: 'Lorem ipsum dolor sit amet, consectetur adipisicing elit. Nesciunt explicabo, quidem dolorum totam consequuntur vitae consectetur, odio asperiores voluptas magni tenetur vel. Accusamus ut, minima quas aliquid consectetur cumque molestiae at modi fugit dolores ipsum, perferendis a quam saepe et doloremque quae reiciendis id, assumenda rerum. Fuga voluptatum earum aliquam?',
+    image: document.createElement('IMG'),
+    image_src: document.getElementById('project_img1').src,
```

Comment:
* As stated in the[ project requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/8_popup.md#:~:text=For%20each%20project%20you%20need%20to%20store%20the%20following%20pieces%20of%20data%20in%20a%20JavaScript%20object%2C%20at%20a%20minimum%3A%20name%2C%20description%2C%20featured%20image%2C%20technologies%2C%20link%20to%20live%20version%2C%20link%20to%20source.) images for the popup should be stored in the javascript object, Kindly remove all hard-coded data from your index file

Reply:
- [ ] @Mylo16 :wave: You have implemented a nice alternative to this, and that's being impressively creative :clap: However, the reviewer's suggestion is a major requirement for this project. Kindly add the `image` as suggested. Here's a [template](https://github.com/olawale-o/portfolio-mobile-version/pull/15/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051R95) that I believe will give you more understanding of what is expected.

- [ OPTIONAL] The pop-modal and the entire project section could be made more dynamic with JavaScript. This will really enhance the performance and loading time of your project on myriad os devices. A quick glimpse at the resource/repo I provided will be a motivation. :rocket: